.husky/pre-commit

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run test-silently
+npm run typescript
+npm run lint
+npm run check-languages
```

Comment:
mad-expo-core also has a husky config inside package.json. Remove one of them. If you decide to keep the old one, consider modifying it to use npm instead of yarn, since we don't use yarn in mad-expo-core

Reply:
Removed the old one